### PR TITLE
Add Python3.9 support

### PR DESCRIPTION
**Pull request by k0t3n**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 
 Mypy stubs for [DRF 3.12.x](https://pypi.org/project/djangorestframework/).
-Supports Python 3.6, 3.7 and 3.8.
+Supports Python 3.6, 3.7, 3.8 and 3.9.
```
Comment (Goldziher) [inline README.md:11]: Maybe just Python 3.6+ ?

**Review by Goldziher — APPROVED** [review]
Nice, thanks!

File: README.md (hunk 1 above)
Comment (k0t3n) [reply README.md:11]: I would rather suggest Python 3.6-3.9. Anyway there is nothing wrong with explicitly listing versions (see DRF's [readme](https://github.com/encode/django-rest-framework#requirements))

File: .travis.yml [hunk 2]
```diff
@@ -18,19 +22,23 @@ jobs:
       script: python ./scripts/typecheck_tests.py
 
     - name: Lint using pre-commit
-      python: 3.8
+      python: 3.9-dev
```
Comment (sobolevn) [inline .travis.yml:25]: Why `-dev`?

**Review by sobolevn — COMMENTED** [review]
Thanks a lot!

File: .travis.yml (hunk 2 above)
Comment (k0t3n) [reply .travis.yml:25]: As far as I can see Travis CI still hasn't added support for stable release ([issue](https://travis-ci.community/t/python-3-9-dev-support/6703) and [versions list](https://docs.travis-ci.com/user/languages/python/#python-versions)).

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thanks!

File: .travis.yml (hunk 2 above)
Comment (MarcinWieczorek) [reply .travis.yml:25]: That list contains 3.9 now.

Comment (sobolevn) [reply .travis.yml:25]: @MarcinWieczorek please, send one more PR 🙂

**Pull request merged**